staging/src/kubevirt.io/containerized-data-importer-api/go.mod

```diff
@@ -3,9 +3,11 @@ module kubevirt.io/containerized-data-importer-api
 go 1.19
 
 require (
+	github.com/kubernetes-csi/external-snapshotter/client/v6 v6.0.1
```

Comment:
Can we get rid of this somehow?

Reply:
Let's keep this go.mod lean and mean.  Copying string consts is fine

Reply:
done